### DISCUSSION TITLE
markdown not supported from 1.6.6 onwards

### DISCUSSION THREAD

**Discussion by TimoNewton** [flyerhq/flutter_chat_ui#615] [Q&A] [answered]
I saw in the history that standard markdown format used to be supported, per MR 206.  However, starting with version 1.6.6 the pattern matching for at least bold and italic were changed so that they no longer match the standard.

Is there some other regex that is being followed for this type of standard?

- bold in markdown is specified with a double asterisk or a double underline.
- italic in markdown is specified by a single asterisk or underline.
- bold and underline is demarked by a triple asterisk or underline.

**Comment by demchenkoalex** [comment, marked as answer]
v2 uses flutter_markdown https://github.com/flyerhq/flutter_chat_ui/issues/646
